var/spack/repos/builtin/packages/procps/libintl-4.0.1.patch

```diff
@@ -0,0 +1,29 @@
+--- a/Makefile.am	2022-10-20 04:58:25.000000000 -0500
++++ b/Makefile.am	2023-06-29 16:38:47.941981053 -0500
+@@ -1,7 +1,6 @@
+ 
+-CYGWINFLAGS =
++CYGWINFLAGS = $(LTLIBINTL)
```

Comment:
It's not the line numbers (`patch` is pretty good if only the line numbers differ), it's the context: `kill.1` -> `man/kill.1`, `kill_SOURCES` -> `src_kill_SOURCES`, etc.

We could use `FileFilter`, but I'm inclined to keep the patch format because of https://gitlab.com/procps-ng/procps/-/merge_requests/189, and the fact that there shouldn't be any more patches.

Reply:
Note that https://gitlab.com/procps-ng/procps/-/merge_requests/189 was merged with minor changes and should be included in `procps` 4.0.4.